### ISSUE TITLE
Not able to hot reload flutter app.

### ISSUE THREAD

**Issue by nilesh14** [flutter/flutter#62131]
I have started a new Flutter default app which is the counter app and when I try run the app on emulator, it installs perfectly but I cannot just change anything and hot reload, I have to reinstall whole app.

Run window is stuck at 

Launching lib\main.dart on sdk gphone x86 in debug mode...
Running Gradle task 'assembleDebug'...
√ Built build\app\outputs\apk\debug\app-debug.apk.
Installing build\app\outputs\apk\app.apk...
This is taking longer than expected...

Although the App gets installed properly and I can increment the counter too. But hot reload is grayed out since I believe Android studio is in disconnected state with the app. So to see the change, I always have to reinstall the whole app which is tedious process. Please guide me in finding what is wrong here.

## Steps to Reproduce

1 - In Android Studio, start a new Flutter application.
2 - Start emulator with API level 30, Android 10.0 + Google Play
3 - Run the app.

## Logs

```
**Run toolbar log:**
Launching lib\main.dart on sdk gphone x86 in debug mode...
Running Gradle task 'assembleDebug'...
√ Built build\app\outputs\apk\debug\app-debug.apk.
Installing build\app\outputs\apk\app.apk...
This is taking longer than expected...
```
**Event log toolbar log:**

7/23/2020
8:10 PM	Can't bind to local 8600 for debugger

```
```

## Flutter Doctor

```
[√] Flutter (Channel stable, v1.17.5, on Microsoft Windows [Version 10.0.14393], locale en-US)
    • Flutter version 1.17.5 at D:\Softwares\flutter
    • Framework revision 8af6b2f038 (3 weeks ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4


[√] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at D:\AndroidSDK
    • Platform android-30, build-tools 30.0.1
    • ANDROID_HOME = D:\AndroidSDK
    • Java binary at: D:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.0)
    • Android Studio at D:\Program Files\Android\Android Studio
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[!] IntelliJ IDEA Community Edition (version 2019.2)
    • IntelliJ at D:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.2
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[√] VS Code, 32-bit edition (version 1.7.2)
    • VS Code at C:\Program Files (x86)\Microsoft VS Code
    • Flutter extension version 3.7.1

[√] Proxy Configuration
    • HTTP_PROXY is set
    • NO_PROXY is localhost,127.0.0.1,LOCALHOST
    • NO_PROXY contains 127.0.0.1
    • NO_PROXY contains localhost

[√] Connected device (1 available)
    • sdk gphone x86 • emulator-5554 • android-x86 • Android 11 (API 30) (emulator)

! Doctor found issues in 1 category.
```

![image](https://user-images.githubusercontent.com/6213713/88302238-3b42bb80-cd23-11ea-91ea-6ded2ae68fd7.png)

```
```

**Comment by TahaTesser** [comment]
Hi @nilesh14 
Can you please try this [solution(https://stackoverflow.com/a/32507892)

If the problem persist after solution, Can you please provide  your `flutter run --verbose`
Thank you

**Comment by nilesh14** [comment]
Hello @TahaTesser  Thank you for responding. I had found this solution earlier and had tried it too before posting my query. Nothing worked at the time.

I checked yesterday and it started working again.. To get flutter to connect to the android emulator I had set some environment variable as below:

HTTP_PROXY=http://127.0.0.1:1087
HTTPS_PROXY=http://127.0.0.1:1087
NO_PROXY=localhost,127.0.0.1,LOCALHOST 

This did not work too so I removed all these variables as it was causing some issue in building the project. I was getting some connection timed out in `flutter pub get` command.
But the next day when I turned on the PC, things became normal again.

Although instant/hot reload working fine, I still get the **Flutter is taking longer than expected to report its views. Still trying...** message in the Run toolbar. 

Launching lib\main.dart on AOSP on IA Emulator in debug mode...
[!] Your app isn't using AndroidX.
    To avoid potential build failures, you can quickly migrate your app by following the steps on https://goo.gl/CP92wY.
Running Gradle task 'assembleDebug'...
√ Built build\app\outputs\apk\debug\app-debug.apk.
Installing build\app\outputs\apk\app.apk...
Flutter is taking longer than expected to report its views. Still trying...
Debug service listening on ws://127.0.0.1:52592/W5IsDKJ_8qA=/ws
Syncing files to device AOSP on IA Emulator...

Thank you
